compiler/src/iree/compiler/Preprocessing/Common/Passes.td

```diff
@@ -10,13 +10,13 @@
 include "mlir/Pass/PassBase.td"
 
 def ConvertConv2DToImg2Col :
-    Pass<"iree-flow-convert-conv2d-to-img2col", ""> {
+    Pass<"iree-convert-conv2d-to-img2col", ""> {
   let summary = "Convert linalg convolution ops to matmul img2col based implementation";
   let constructor = "mlir::iree_compiler::IREE::createConvertConv2DToImg2ColPass()";
 }
 
 def PadLinalgOps :
-    Pass<"iree-flow-pad-linalg-ops", ""> {
+    Pass<"iree-pad-linalg-ops", ""> {
```

Comment:
we'll still want to prefix these to avoid polluting the global namespace - maybe `iree-preprocessing-...`?

Reply:
sounds good. Will change before landing.